dotnetv3/dynamodb/scenarios/PartiQL_Batch_Scenario/PartiQL_Batch_Scenario/PartiQLBatchMethods.cs

```diff
@@ -120,7 +120,7 @@ public static async Task<bool> GetBatch(
             int year1,
             int year2)
         {
-            var getBatch = $"SELECT FROM {tableName} WHERE title = ? AND year = ?";
+            var getBatch = $"SELECT * FROM {tableName} WHERE title = ? AND year = ?";
```

Comment:
The PartiQLBatch program threw an exception: 

{"The given key 'title' was not present in the dictionary."}

You may want to look at that. Make sure it runs successfully. Other then that , i saw the updates to the code and all other programs ran successfully